### PR TITLE
Added IP-Address Tracker React js Project

### PR DESCRIPTION
**Pull request by Maheshwari-Love**
Closes: #498 

- **Title** : IP-Address Tracker
- **Name:** : Maheshwari Love
- **Idenitfy yourself:** :  GSSOC

### Describe the add-ons or changes you've made 📃

Added the Ip-address-Tracker Reackjs application

### How Has This Been Tested? ⚙️

I have run it on my local system and have tested.

### Screenshots 📷

image

### Type of change ☑️

What sort of change have you made:
Added new project 

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by Maheshwari-Love** [comment]
@Avdhesh-Varshney please review this pR

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
follow & read the guidelines properly

**Comment by Maheshwari-Love** [comment]
> follow & read the guidelines properly

@Avdhesh-Varshney requested changes done

File: React-JS-Projects/Advanced/IP-Address-Tracker/README.md
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Advanced/IP-Address-Tracker/README.md:1]: @Maheshwari-Love remove your commented part and fix the ss

image

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
carefully follow the guidelines. there are some more changes are also left.

**Review by Avdhesh-Varshney — APPROVED** [review]

**Pull request merged**